mne/viz/epochs.py

```diff
@@ -69,6 +70,8 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    logscale : bool 
+        Plots logarithmic colorscale (base 10) of epochs if True.
```

Comment:
I would say base 2 is more relevant, because triggers are generally coded
in a binary base

On 3 January 2016 at 19:09, Mainak Jas <notifications@github.com> wrote:

> In mne/viz/epochs.py
> <https://github.com/mne-tools/mne-python/pull/2763#discussion_r48695403>:
>
> > @@ -69,6 +70,8 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
> >          Figure instance to draw the image to. Figure must contain two axes for
> >          drawing the single trials and evoked responses. If None a new figure is
> >          created. Defaults to None.
> > +    logscale : bool
> > +        Plots logarithmic colorscale (base 10) of epochs if True.
>
> Do we want base 10 as default @kingjr <https://github.com/kingjr> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2763/files#r48695403>.
>